changelog/fragments/1677000534-Restore-support-for-memcached-metrics-inputs.yaml

```diff
@@ -0,0 +1,27 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Restore support for memcached/metrics inputs.
+
+# Affected component; a word indicating the component this changeset affects.
+component: Elastic agent.
```

Comment:
Is this a field that takes a freeform value?  I ask because I'm seeing `agent` and `spec` in some of the other [changelog fragment files](https://github.com/elastic/elastic-agent/tree/main/changelog/fragments).

Reply:
It isn't standardized. I believe this was copied from the Elasticsearch version of this tool. I'm not sure it even makes sense for the agent repository (it doesn't have sub-components really), but it will when we start using it in Beats.